AutomatedTesting/Gem/PythonTests/scripting/FileMenu_Default_NewAndOpen.py

```diff
@@ -4,25 +4,26 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
-from PySide2 import QtWidgets
 
-import editor_python_test_tools.pyside_utils as pyside_utils
+from PySide2 import QtWidgets
+from editor_python_test_tools.utils import TestHelper as helper
 from editor_python_test_tools.utils import Report
-
 import azlmbr.legacy.general as general
+import editor_python_test_tools.pyside_utils as pyside_utils
 
 
 # fmt: off
 class Tests():
-    new_action  = "File->New action working as expected"
-    open_action = "File->Open action working as expected"
-# fmt: on
 
+    initialTabTestResults = "Verified no tabs open"
+    newTabTestResults = "New tab opened successfully"
+    fileMenuTestResults = "Open file window triggered successfully"
 
-GENERAL_WAIT = 0.5  # seconds
-
+# fmt: on
+TIME_TO_WAIT = 3
```

Comment:
Nit: Add a line here to separate the text formatting statement from actual code.

Reply:
fmt line removed per @smurly 's suggestion and left a line there for spacing.